change the folder structure of each dataset.

xyang41 opened the pull request:
Each dataset should have 3 folders, named `00-raw`, `01-change-fields` and `02-matrix` respectively. `00-raw` contains the raw data, and different datasets may be stored in different formats. `01-change-fields` contains the transferred data, automatically transformed from `00-raw` by `transfer.py`, which should be **designed based on your dataset**. And `02-matrix` includes the final processed version, which adds additional necessary features to `01-change-fields` by given `process.py`, and will be directly inputted to UniRE model.

xyang41 committed:
change the folder structure of each dataset.

ybwu commented on config.yml:
```diff
@@ -1,9 +1,9 @@
-train_file: train.json
-dev_file: dev.json
-test_file: test.json
+train_file: 02/train.json
```
目录的命名除了数字以外可以给一个简单的名字: 00-raw 表示生语料， 01-change-fields 表示格式转化成符合unire的json格式(用 transfer.py)， 02-matrix 表示增加joint matrix fields. 可以参考[这里](https://github.com/AntNLP/antnlp-tproj/tree/main/data).

ybwu replied:
不同数据集的config.yml应该放在哪里？

ybwu commented on data/ACE2004/ace2004.sh:
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+mkdir -p fold1/01
```
这里目录名需修改。 或者目录先建立好，不用脚本生成。ace2004.sh只负责[调用](https://github.com/AntNLP/antnlp-tproj/tree/main/data)每个目录下的脚本

ybwu commented on data/ACE2005/ace2005.sh:
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+mkdir -p 01  # dir of the transferred version
```
目录名

ybwu commented on data/README.md:
```diff
@@ -1,12 +1,17 @@
 
+### Dataset Folder Format
+Each dataset should have 3 folders, named `00`, `01` and `02` respectively. 
+`00` contains the raw data, and different datasets may be stored in different formats. 
```
修改目录名称

ybwu commented on data/SciERC/scierc.sh:
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+mkdir -p 01 # dir of the transferred version
```
目录名称

xyang41 committed:
modified:   README.md
	modified:   config.yml
	modified:   data/ACE2004/ace2004.sh
	modified:   data/ACE2005/ace2005.sh
	modified:   data/README.md
	modified:   data/SciERC/scierc.sh
